types/node/buffer.buffer.d.ts

```diff
@@ -160,6 +160,7 @@ declare module "buffer" {
              * @param encoding The encoding of `string`. **Default:** `'utf8'`.
              */
             from(string: WithImplicitCoercion<string>, encoding?: BufferEncoding): Buffer<ArrayBuffer>;
+            from(arrayOrString: WithImplicitCoercion<ArrayLike<number> | string>): Buffer<ArrayBuffer>;
```

Comment:
FWIW these declarations are missing their JSDoc comments. Probably not intended but IIRC we won't render the "wrong" one from above onto this overload?

Reply:
It seems the JSDoc for the first overload is rendered.
![image](https://github.com/user-attachments/assets/86d9b66c-87ff-44ed-a6aa-d941ab6e9325)
I removed it for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72097#discussion_r1977016054, but I can add it  for the new overload.